test/test_utils.go

```diff
@@ -186,7 +202,14 @@ func ReadAndAssertDelete(
 	})
 }
 
+var configureDump = sync.OnceFunc(func() {
+	dump.Config(dump.WithoutPosition())
+})
+
 func IsDataEqual(is *is.I, a, b sdk.Data) {
```

Comment:
All of this is quite elaborate. Can't this simple be done with [`google/go-cmp`](https://github.com/google/go-cmp)?

```
is.Equal("", cmp.Diff(a, b))
```

Reply:
Yup, could be done, and damn me not knowing about the existence of that package.